eng/pipelines/templates/jobs/ci.yml

```diff
@@ -83,17 +97,24 @@ jobs:
         condition: and(succeeded(), eq(variables['SetDevVersion'],'true'))
         displayName: Apply version settings to repository
 
+      - task: PythonScript@0
```

Comment:
Is there any reason these all steps next to each other? Is there an ordering issue which requires the gap?

Reply:
Do you mean why is daily-dev-build in between these two steps? I can't recall whether this was an artifact of the order in which I edited things or not. I can't find any dependencies across steps that would prevent me from moving the dev build step lower down.

Reply:
Exactly I'd like to keep all the sparse checkout steps next to each other if possible.